test/lsp.py

```diff
@@ -180,6 +177,10 @@ def __init__(self):
         args = create_cli_parser().parse_args()
         self.solc_path = args.solc_path
         self.project_root_dir = os.path.realpath(args.project_root_dir) + "/test/libsolidity/lsp"
+        if os.name == 'nt':
```

Comment:
What about using `as_uri()` from `pathlib`?

Reply:
I did not know about that. will be trying.